src/dhcp_mon.cpp

```diff
@@ -375,6 +375,13 @@ int dhcp_mon_init(int window_sec, int max_count, int db_update_interval)
         dhcp_unhealthy_max_count = max_count;
         db_update_interval_sec = db_update_interval;
 
+        // Set clear_counter_timeout based on db_update_interval
+        if (db_update_interval < 4) {
+            clear_counter_timeout = 5;
```

Comment:
Can you define constant for these magic numbers? 

```
constant int MINIMAL_CLEAR_COUNTER_TIMEOUT_SEC = 5;
constant int CLEAR_COUNTER_DELAY_AFTER_DB_UPDATE_SEC = 1;
```

Reply:
Fixed